Sources/Build/describe().swift

```diff
@@ -128,6 +128,11 @@ public func describe(prefix: String, _ conf: Configuration, _ modules: [Module],
     #if os(Linux)
          args += ["-fPIC"]
     #endif
+    #if os(OSX)
+        if let sysroot = Toolchain.sysroot {
```

Comment:
By the way using `#if`s here is a slippery slope that would make it hard to teach `swift-build` to cross-compile.